couchbase-on-ubuntu/azuredeploy.json

```diff
@@ -69,27 +69,27 @@
     },
     "cbPackageDownloadBase": {
       "type": "string",
-      "defaultValue": "http://packages.couchbase.com/releases/3.0.3/",
+      "defaultValue": "http://packages.couchbase.com/releases/4.1.1/",
       "metadata": {
         "description": "Couchbase package download location"
       }
     },
     "cbPackage": {
       "type": "string",
-      "defaultValue": "couchbase-server-enterprise_3.0.3-ubuntu12.04_amd64.deb",
+      "defaultValue": "couchbase-server-enterprise_4.1.1-ubuntu12.04_amd64.deb",
```

Comment:
Is it possible to assume this based on the value of the cbPackageDownloadBase parameter above? Look into allowedValues if you need to control user input https://github.com/Azure/azure-quickstart-templates/blob/master/checkpoint-single-nic/azuredeploy.json#L29

Also the above link will also show you how to offer more than one OS or couchbase version in case you want to make this template install couchbase on RHEL, CentOs, Debian or Ubuntu as shown in the documentation http://developer.couchbase.com/documentation/server/current/getting-started/installing.html. Let me know if you need more info on this, I can help.